www/grpc/blockchain.go

```diff
@@ -233,7 +234,7 @@ func (s *blockchainServer) GetPublicKey(_ context.Context,
 	return &pactus.GetPublicKeyResponse{PublicKey: publicKey.String()}, nil
 }
 
-func validatorToProto(val *validator.Validator) *pactus.ValidatorInfo {
+func validatorToProto(val *validator.Validator, sf state.Facade) *pactus.ValidatorInfo {
```

Comment:
@izikdepth 

you can make it a member function, like:
```
func (s *blockchainServer) validatorToProto(val *validator.Validator) *pactus.ValidatorInfo {
...
}
```

Then you can access to `state` like `s.state`

Reply:
Please do the same for: `accountToProto` and `voteToProto`

Reply:
is this correct?

```func (s *blockchainServer) validatorToProto(val *validator.Validator) *pactus.ValidatorInfo {
	data, _ := val.Bytes()
	return &pactus.ValidatorInfo{
		Hash:                val.Hash().Bytes(),
		Data:                data,
		PublicKey:           val.PublicKey().String(),
		Address:             val.Address().String(),
		Number:              val.Number(),
		Stake:               val.Stake(),
		LastBondingHeight:   val.LastBondingHeight(),
		LastSortitionHeight: val.LastSortitionHeight(),
		UnbondingHeight:     val.UnbondingHeight(),
		AvailabilityScore:   float32(s.state.AvailabilityScore(val.Number())),
	}
}

func (s *blockchainServer) accountToProto(addr crypto.Address, acc *account.Account) *pactus.AccountInfo {
	data, _ := acc.Bytes()
	return &pactus.AccountInfo{
		Hash:    acc.Hash().Bytes(),
		Data:    data,
		Number:  acc.Number(),
		Balance: acc.Balance(),
		Address: addr.String(),
	}
}

func (s *blockchainServer) voteToProto(v *vote.Vote) *pactus.VoteInfo {
	cpRound := int32(0)
	cpValue := int32(0)
	if v.IsCPVote() {
		cpRound = int32(v.CPRound())
		cpValue = int32(v.CPValue())
	}

	return &pactus.VoteInfo{
		Type:      pactus.VoteType(v.Type()),
		Voter:     v.Signer().String(),
		BlockHash: v.BlockHash().Bytes(),
		Round:     int32(v.Round()),
		CpRound:   cpRound,
		CpValue:   cpValue,
	}
}```

Reply:
@izikdepth something like that:
```go
func (bs *blockchainServer) validatorToProto(val *validator.Validator) *pactus.ValidatorInfo {
	data, _ := val.Bytes()
	return &pactus.ValidatorInfo{
		Hash:                val.Hash().Bytes(),
		Data:                data,
		PublicKey:           val.PublicKey().String(),
		Address:             val.Address().String(),
		Number:              val.Number(),
		Stake:               val.Stake(),
		LastBondingHeight:   val.LastBondingHeight(),
		LastSortitionHeight: val.LastSortitionHeight(),
		UnbondingHeight:     val.UnbondingHeight(),
                AvailabilityScore:   float32(bs.state.AvailabilityScore(val.Number())),
	}
}

```